http/src/client/mod.rs

```diff
@@ -160,9 +126,25 @@ impl Debug for State {
 /// `client`.
 ///
 /// [here]: https://discord.com/developers/applications
-#[derive(Clone, Debug)]
+#[derive(Debug)]
 pub struct Client {
```

Comment:
should we sort this struct while we're at it? i noticed some other structs have been sorted

Reply:
Done.